package.json

```diff
@@ -10,15 +10,15 @@
     "type": "git",
     "url": "https://github.com/sourcegraph/sourcegraph-vscode.git"
   },
+  "enableProposedApi": true,
```

Comment:
in launch.json we are specifying `--enable-proposed-api` and we're also specifying it here. Which one is actually required for us today?